gradle/libs.versions.toml

```diff
@@ -33,6 +34,7 @@ apache-commons-lang3 = { module = "org.apache.commons:commons-lang3", version =
 neo4j-ogm-core = { module = "org.neo4j:neo4j-ogm-core", version.ref = "neo4j"}
 neo4j-ogm-bolt-driver = { module = "org.neo4j:neo4j-ogm-bolt-driver", version.ref = "neo4j"}
 neo4j-driver = { module = "org.neo4j.driver:neo4j-java-driver", version.ref = "neo4j5"}
+netty-handler = { module = "io.netty:netty-handler", version.ref = "netty"}
```

Comment:
You mentioned ini4j in the description? But its for neo4j, correct?

Reply:
Yes, sorry, that was a mistake from my side. I will update the description